README.md

```diff
@@ -1,43 +1,43 @@
-Reviewlette
-===========
-
-[![Coverage Status](https://img.shields.io/coveralls/jschmid1/reviewlette.svg)](https://coveralls.io/r/jschmid1/reviewlette)
-[![Code Climate](https://codeclimate.com/github/jschmid1/reviewlette.png)](https://codeclimate.com/github/jschmid1/reviewlette)
-
-Tool to automatically assign a "Reviewer" to a GitHub Issue and to the attached Trello Card.
-
+# Reviewlette
+Tool to automatically assign reviewers to GitHub pull requests and to move and comment on their Trello cards.
 
 What it does:
 
-- Finds unassigned issues on GitHub.
-- Assigns a member of your team.
-- Locates the right Card on Trello.
-- Checks if the assignee is on vacation(using tel).
-- Adds the assigned member to the Card.
-- Move the card to 'In review'
-
+- Finds pull requests with missing reviewers in your GitHub repos.
+- Assigns random members of your team.
+- Takes vacations of team members into account (using tel).
+- Locates the right card in your Trello board.
+- Mentions the assigned reviewer in a comment on the card.
+- Moves the card to the 'In review' column.
 
 ## Installation
+For the latest and greatest version you should `git clone https://github.com/SUSE/reviewlette`
 
-```
-gem install reviewlette
-reviewlette
-```
+## Setup
+Copy the example config files (e.g. `cp ./config/members_example.yml ./config/members.yml)` and fill them out.
 
+## Usage
+Regulary run `./bin/reviewlette` (e.g. with a cronjob or a systemd timer) to check for new or changed pull requests.
 
-## Setup
+### Matching Trello cards
+To match a Trello card to a pull request, its title has to end with the card number (not the id)
 
-#### Name your pullrequest like so:
-#### Review_#23_name_of_review_42  <= trello card number
+#### Example:
 
-Fill `config/.trello.yml` (instructions in the file)
-Fill `config/.github.yml` (instructions in the file
+URL of the Trello card: _https://trello.com/c/cardid/4242-fix-everything_
 
+Pull request title should be: `Fix almost everything 4242`
 
----
+__Note:__ Pull requests without a matching Trello card get skipped and won't be assigned to a reviewer.
 
-[Using Octokit as a GitHub api wrapper](https://github.com/octokit/octokit.rb)
-[Using ruby-trello as a Trello api wrapper](https://github.com/jeremytregunna/ruby-trello)
+### Flags
+You can add flags to your pull request title to tell Reviewlette. Just put a pair of brackets in front of your pull request title.
```

Comment:
Maybe this will work better?

```
You can tweak Reviewlette behavior with additional flags in the PR title. Flags should be passed in square brackets, at the beginning of the PR name.